specification/2.0/Specification.adoc

```diff
@@ -146,91 +269,146 @@ The following are outside the scope of the initial design of glTF:
+* https://github.com/KhronosGroup/glTF/blob/master/extensions/2.0/Khronos/KHR_texture_basisu/README.md[KHR_texture_basisu] for GPU-compatible block-compressed textures.
+
+Future extensions **MAY** include more compression methods including those for animation data.
 
 
 [[versioning]]
-== Versioning
+== Versioning (Informative)
 
-Any updates made to glTF in a minor version will be backwards and forwards compatible. Backwards compatibility will ensure that any client implementation that supports loading a glTF 2.x asset will also be able to load a glTF 2.0 asset. Forwards compatibility will allow a client implementation that only supports glTF 2.0 to load glTF 2.x assets while gracefully ignoring any new features it does not understand.
+This section is marked as informal as there is no binding responsibility on implementations of the glTF — these guarantees are however a contract between the 3D Formats Working Group and developers using this Specification.
 
-A minor version update can introduce new features but will not change any previously existing behavior. Existing functionality can be deprecated in a minor version update, but it will not be removed.
+Any updates made to glTF in a minor version **MUST** be backwards and forwards compatible. Backwards compatibility means that any client implementation that supports loading a glTF 2.x asset will also be able to load a glTF 2.0 asset. Forwards compatibility means that a client implementation that only supports glTF 2.0 is able to load glTF 2.x assets while gracefully ignoring any new features it does not understand.
 
-Major version updates are not expected to be compatible with previous versions.
+A minor version update **MAY** introduce new features but **MUST NOT** change any previously existing behavior. Existing functionality **MAY** be deprecated in a minor version update, but it **MUST NOT** be removed.
 
+Major version updates **MAY** be incompatible with previous versions.
 
-[[file-extensions-and-mime-types]]
-== File Extensions and MIME Types
 
-* `*.gltf` files use `model/gltf+json`
-* `*.bin` files use `application/octet-stream` or `application/gltf-buffer`
-  - When using MIME type `application/gltf-buffer`, the binary file extension may be `*.bin`, `*.glbin`, or `*.glbuf`.  See the https://www.iana.org/assignments/media-types/application/gltf-buffer[gltf-buffer registration].
-* Texture files use the official `image/*` type based on the specific image format. For compatibility with modern web browsers, the following image formats are supported: `image/jpeg`, `image/png`.
-+
-[NOTE]
-.Implementation Note
-====
-Implementations should use the image type pattern matching algorithm from the https://mimesniff.spec.whatwg.org/#matching-an-image-type-pattern[MIME Sniffing Standard] to detect PNG and JPEG images as file extensions may be unavailable in some contexts.
-====
+[[file-extensions-and-media-types]]
+== File Extensions and Media Types
```

Comment:
I understand why the use of a `.gltf` extension for JSON glTF files is not required — we don't want to prohibit use cases like REST APIs, or perhaps like `.vrm`.

But, is it worth including some language that normatively prohibits a JSON glTF file using a `.glb` prefix, or a binary GLB file using `.gltf`? Such mixups would be unfortunate.